### PR TITLE
feat: ask-cli v2 compatibility + new “manifest” & “extensions” cmds

### PR DESCRIPTION
**Pull request by amzn-chris-subagio**
This makes the `litexa deploy` cli command compatible with the new ask-cli v2

bonus features that came up during the development:
* a new `litexa manifest` command that prints out the generated manifest
* a new `litexa extensions` command that prints out the list of active extensions

bugfixes:
* fixes the bug where each smapi log was resetting the manifest log file
* manifest deployment now passes up string messages meant for cli users

tested with test suite and creation/deployment of a test skill against:
* ask-cli 1.7.23
* ask-cli 2.10.2

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa-games/litexa/issues
[license]: https://www.apache.org/licenses/LICENSE-2.0
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

**Comment by ayrc** [comment]
Addresses https://github.com/alexa-games/litexa/issues/167

File: packages/litexa/src/command-line/deploy.coffee [hunk 1]
```diff
@@ -113,44 +120,50 @@ module.exports.run = (options) ->
     if deploymentTypes.assets
       assetsLogger = new LoggingChannel({
         logPrefix: 'assets'
-        logStream
-        logFile: path.join(deployRoot,'assets.log')
+        logSteam: logger.logStream
```
Comment (joseifert) [inline packages/litexa/src/command-line/deploy.coffee:123]: logSteam typo > applies to below as well.

File: packages/litexa-deploy-aws/test/manifest.spec.coffee [hunk 2]
```diff
@@ -20,11 +20,23 @@ chai.use(chaiAsPromised)
 
 
 describe 'construct and deploy artifacts for manifest', ->
+  this.timeout(5000)
```
Comment (joseifert) [inline packages/litexa-deploy-aws/test/manifest.spec.coffee:23]: Maybe add a comment as to what this is for?

File: packages/litexa/src/command-line/deploy.coffee (hunk 1 above)
Comment (amzn-chris-subagio) [reply packages/litexa/src/command-line/deploy.coffee]: good catch! Fixed

File: packages/litexa-deploy-aws/test/manifest.spec.coffee (hunk 2 above)
Comment (amzn-chris-subagio) [reply packages/litexa-deploy-aws/test/manifest.spec.coffee:24]: done

**Pull request merged**